ui/lib/core/addon/components/search-select.js

```diff
@@ -110,6 +110,10 @@ export default class SearchSelect extends Component {
   }
 
   formatInputAndUpdateDropdown(inputValues) {
+    // For model attributes using editType: 'searchSelect' where the API returns a string,
+    // we need to wrap the value in an array to match the format expected by the dropdownOptions.
+    // Example: model/transform.js -> template
+    inputValues = Array.isArray(inputValues) ? inputValues : [inputValues];
```

Comment:
Can we add a test for this behavior? Search select is a pretty bloated component as is, so I'm nervous to add even seemingly small logic to it.

Reply:
Test coverage is incoming. It's a mess in there, so I stayed with selector patterns that existed and added coverage for search select but also acceptance test coverage for the create/edit workflow.